### DISCUSSION TITLE
ArrayGetValuesByIndexArray new op

### DISCUSSION THREAD

**Discussion by kikohs** [cables-gl/cables_docs#493] [Feature Requests] [answered]
Handy to fetch values from array at many indices at once. 

https://dev.cables.gl/op/Ops.User.kikohs.ArrayGetValuesByIndexArray

Let me know if you want to add it to core.

**Comment by steam0r** [comment, marked as answer]
we added this on dev.cables.gl and it will be in the next release, thanks for your contribution!
